src/_pytest/outcomes.py

```diff
@@ -225,7 +225,7 @@ def importorskip(
         warnings.simplefilter("ignore")
         try:
             __import__(modname)
-        except ImportError as exc:
+        except ModuleNotFoundError as exc:
```

Comment:
Let's add a new parameter to this function, `*, exc_type: Type[ImportError] = ModuleNotFoundError` and use it here:

```suggestion
        except exc_type as exc:
```

This way we provide a escape hatch to users which rely on the old behavior.

Also please update the docs, describing the parameters and explicitly showing an example for users that want the old behavior.